invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/icon.overrides

```diff
@@ -3,7 +3,8 @@
 ***********************************************/
 
 .inline-id-icon {
```

Comment:
Note: I did not make the name refactoring as requested in the original PR by @kpsherva because this broke more thing accross modules, e.g.

![Screenshot 2022-05-20 at 10 20 00](https://user-images.githubusercontent.com/6756943/169486814-8a0fb37a-f3fb-4652-a53e-3f40d7578328.png)

I think that name refactor can be done as a separate task if needed